addons/wowl/static/src/components/error_dialogs/error_dialogs.ts

```diff
@@ -52,9 +52,13 @@ export class ErrorDialog extends Component<ErrorDialogProps, OdooEnv> {
     // if (self.browserDetection.isBrowserChrome()) {  --> add to OdooBrowser? We should check things since now Edge uses chromium do they have same behavior?
     //  traceback = ev.reason.stack;
     // } else {
-    this.traceback = `${this.env._t("Error:")} ${message}\n${stack}`;
```

Comment:
I would not remove that

Reply:
Are you sure? Because in web, we don't display "Error" in the traceback. It is only added when you use the clipboard.

Reply:
ok then